Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml

```diff
@@ -62,8 +62,8 @@
       input:
         !type:CableDeviceNode
         nodeGroupID: HVPower
-  - type: ApcPowerReceiver
-  - type: ExtensionCableReceiver
+  #- type: ApcPowerReceiver
+  #- type: ExtensionCableReceiver
```

Comment:
imo if you want to remove the apc power requirement, you should also remove it from the code of the amecontrollercomp

Reply:
That just makes it more difficult to reintroduce the function later, and the AME Controller's code already accounts for the ApcPowerReceiver being missing.

Reply:
![image](https://user-images.githubusercontent.com/7806367/147275004-d5fb7b00-fb89-4c15-a88a-56fdbee60c84.png)